tpal-backend/src/main/java/com/example/tpalbackend/entities/PostEntity.java

```diff
@@ -0,0 +1,38 @@
+package com.example.tpalbackend.entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Date;
+import java.util.UUID;
+
+@Entity
+@Table(name="posts")
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class PostEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private UUID id;
+    private String title;
+    private String description;
+    private String placeOfDeparture;
```

Comment:
If name is complext somethingThenSomethingElse, add @Column(name  = 'something_then_something_else')